Source/Core/DolphinWX/X11Utils.cpp

```diff
@@ -38,7 +38,12 @@ void ToggleFullscreen(Display *dpy, Window win)
 	// Send the event
 	if (!XSendEvent(dpy, DefaultRootWindow(dpy), False,
 	                SubstructureRedirectMask | SubstructureNotifyMask, &event))
+	{
 		ERROR_LOG(VIDEO, "Failed to switch fullscreen/windowed mode.");
```

Comment:
We don't use the function for X11 anyway, so I think right now just some basic error handling is enough.